core/src/toga/widgets/webview.py

```diff
@@ -58,9 +74,16 @@ def __init__(
 
         self.user_agent = user_agent
 
+        # If URL is allowed by user interaction or user on_navigation_starting
+        # handler, the count will be set to 0
+        self._url_count = 0
```

Comment:
I'm confused why this is a "count". All the uses are a simple boolean, not "counting" anything.

Reply:
OK, changed to boolean